docs/user/filter_expressions.rst

```diff
@@ -222,6 +222,39 @@ filter expressions.
 
    Boolean specifying whether the user has enabled Do Not Track.
 
+.. js:attribute:: normandy.experiments
+
+   Object with several arrays containing the unique slugs for experiments that
+   the user has participated in. Currently, this is limited to preference
+   experiments.
+
+   .. js:attribute:: normandy.experiments.all
+
+      Array of experiment slugs for every experiment that the user has  enrolled
+      in, whether currently active or expired.
+
+   .. js:attribute:: normandy.experiments.active
+
+      Array of experiment slugs for active experiments that the user is enrolled
+      in.
+
+   .. js:attribute:: normandy.experiments.expired
+
+      Array of experiment slugs for expired experiments that the user has
+      enrolled in.
+
+   .. code-block:: javascript
+
+      // Target clients that have ever participated in the "australis"
+      // experiment, including clients that are currently running it
+      "australis" in normandy.experiments.all
```

Comment:
We _could_ add JEXL transforms that call Set methods, and have something like:

```js
normandy.experiments.all|includes("australis")
```

But that would stand out as weird compared to all the other APIs in filter expressions, which use lists and work with the `in` operator. 

The filter expression doesn't really care about enforcing uniqueness or lack of order in the list, so maybe `PreferenceExperiments.getAll()` should return a set, and then the code in `ClientEnvironment` should convert that to a list?

Although it'd be nice to patch JEXL to support `in` for sets. We'd just have to add it here: https://github.com/TechnologyAdvice/Jexl/blob/master/lib/grammar.js#L53